src/views/gitStashExplorer.ts

```diff
@@ -0,0 +1,43 @@
+'use strict';
+import { Event, EventEmitter, ExtensionContext, TreeDataProvider, TreeItem, Uri, window } from 'vscode';
+import { ExplorerNode, StashNode } from './gitExplorerNodes';
+import { GitService, GitUri } from '../gitService';
+import { StashCommitNode } from './stashCommitNode';
+
+export * from './gitExplorerNodes';
+
+export class GitStashExplorer implements TreeDataProvider<ExplorerNode>  {
```

Comment:
Let's just call it StashExplorer -- I know my naming has git everywhere, but I'm going to try to move away from it eventually ;)